docs/docs-network/prerequisites.md

```diff
@@ -122,6 +122,14 @@ nc -zuv [YOUR_EXTERNAL_IP] 40400
 If port forwarding isn't properly configured, your node may not be able to participate in P2P duties.
 :::
 
+:::info Dynamic IPs and cgNAT
+If your ISP assigns a dynamic IP address, you can set `P2P_QUERY_FOR_IP=true` to have your node automatically detect and update its external IP.
```

Comment:
@alexghr is this confirmed to automatically update the peer ip if the dynamic IP changes without restarts?